doc/directives.py

```diff
@@ -53,3 +56,166 @@ def run(self):
+        dirname = os.path.dirname(fname)
+
+        try:
+            if 'intro' in self.options:
+                intro = self.options['intro'][:195] + '...'
+            else:
+                _, blocks = sphinx_gallery.gen_rst.split_code_and_text_blocks(abs_fname)
+                intro, _ = sphinx_gallery.gen_rst.extract_intro_and_title(abs_fname, blocks[0][1])
+
+            thumbnail_rst = sphinx_gallery.backreferences._thumbnail_div(
+                dirname, basename, intro)
+
+            if 'figure' in self.options:
+                rel_figname, figname = env.relfn2path(self.options['figure'])
+                save_figname = os.path.join('_static/thumbs/',
+                                            os.path.basename(figname))
+
+                try:
+                    os.makedirs('_static/thumbs')
+                except OSError:
+                    pass
+
+                x, y = (400, 280)
+                if 'size' in self.options:
+                    x, y = self.options['size'].split(" ")
+
+                sphinx_gallery.gen_rst.scale_image(figname, save_figname,
+                                                   x, y)
+                # replace figure in rst with simple regex
+                thumbnail_rst = re.sub(r'..\sfigure::\s.*\.png',
+                                       '.. figure:: /{}'.format(save_figname),
+                                       thumbnail_rst)
+
+            thumbnail = StringList(thumbnail_rst.split('\n'))
+            thumb = nodes.paragraph()
+            self.state.nested_parse(thumbnail, self.content_offset, thumb)
+
+            return [thumb]
+        except FileNotFoundError as e:
+            print(e)
+            return []
+
+
+
+GALLERY_TEMPLATE = """
+.. raw:: html
+
+    <div class="sphx-glr-thumbcontainer" tooltip="{tooltip}">
+    
+.. only:: html
+
+    .. figure:: /{thumbnail}
+    
+        {description}
+        
+.. raw:: html
+
+    </div>
+"""
+
+
+class CustomGalleryItemDirective(Directive):
+    """Create a sphinx gallery style thumbnail.
+    tooltip and figure are self explanatory. Description could be a link to
+    a document like in below example.
+    Example usage:
+    .. customgalleryitem::
+        :tooltip: I am writing this tutorial to focus specifically on NLP for people who have never written code in any deep learning framework
+        :figure: /_static/img/thumbnails/babel.jpg
+        :description: :doc:`/beginner/deep_learning_nlp_tutorial`
+        :size: put image size here
+    If figure is specified, a thumbnail will be made out of it and stored in
+    _static/thumbs. Therefore, consider _static/thumbs as a 'built' directory.
+    """
+
+    required_arguments = 0
+    optional_arguments = 1
+    final_argument_whitespace = True
+    option_spec = {'tooltip': directives.unchanged,
+                   'figure': directives.unchanged,
+                   'description': directives.unchanged,
+                   'size': directives.unchanged}
```

Comment:
Ah, that's right, I made it so you could set the size explicitly I think...

Reply:
Note: do we want to remove the tooltips? They seem a bit superfluous. If so, it should be sufficient to remove the `tooltip="{tooltip}"` from line 137, the tooltip option from lines 157, 168, and 215, as well as the if statement on lines 178-181